src/Domain/LeanCode.DomainModels.MassTransitRelay/MassTransitRelayModule.cs

```diff
@@ -1,37 +1,27 @@
-using System;
-using System.Reflection;
 using Autofac;
-using GreenPipes;
 using LeanCode.Components;
 using LeanCode.DomainModels.MassTransitRelay.Inbox;
 using LeanCode.DomainModels.MassTransitRelay.Middleware;
 using LeanCode.DomainModels.MassTransitRelay.Outbox;
 using LeanCode.DomainModels.MassTransitRelay.Simple;
 using LeanCode.PeriodicService;
 using MassTransit;
-using MassTransit.AutofacIntegration;
 using Microsoft.Extensions.DependencyInjection;
 
 namespace LeanCode.DomainModels.MassTransitRelay
 {
-    public class MassTransitRelayModule : AppModule
+    public abstract class MassTransitRelayModule : AppModule
```

Comment:
I think the module should _enforce_ calling `AddMassTransit` and just allow to change whats specified inside the closure (or after the call).

Reply:
I don't think it'll be practical. Also, I think we may give client flexibility which container to use (say, have to abstract methods methods, so that they implement one and leave the second one empty). Also, sometimes we may not want to call  plain `AddMassTransit`, e.g. when using MultiBus (and we do use it in one project).